### ISSUE TITLE
Label Template from Dropdown

### ISSUE THREAD

**Issue by craigs100** [imulus/Archetype#254]
Can't set Label Template to be the text of a dropdown selection. Limits editorial use for rapid data selection.

![screenshot2015-01-29](https://cloud.githubusercontent.com/assets/1143883/5965313/1e5ef8ea-a7f3-11e4-9e1c-78d459ae364a.png)

**Comment by rasmuseeg** [comment]
I just tried when its an array, {{links.length}} doesn't work either.

**Comment by kgiszewski** [comment]
TL;DR edition

You have to create your own label template.

Long Version

Archetype doesn't know anything about it's children except it's `model` and `config`.

By default, all Archetype can show is the value in the model. If it turns out to be something like a node Id (1234), it's not very useful to just display `1234`. If it's a JSON object, you get `[object Object]`, again not very useful. 

As Archetype has evolved, I have added several label templates that recognize a limited set of core property editor types. Dropdown is not one of them.

To bridge the gap, you can implement your own label editor like so: https://github.com/kgiszewski/ArchetypeManual/blob/master/02%20-%20Configuration.md#label-template

For more core items to be added, it would have to be a feature request or via a pull request.

Hope that clears up why it doesn't work (but you can make it work).

RTE, UrlPicker and MNTP are the only built-in at this time.
